docs/components/board/beaglebone.md

```diff
@@ -74,3 +74,23 @@ Configuring these attributes on your board allows you to integrate [digital inte
 ### `digital_interrupts`
 
 {{< readfile "/static/include/components/board/board-digital-interrupts.md" >}}
+
+## Next steps
```

Comment:
Duplicated; not sure which one you want to use. If the second one, why link to board API twice (as with base etc)?